modules/ocf_stats/manifests/grafana.pp

```diff
@@ -0,0 +1,62 @@
+# grafana config
+class ocf_stats::grafana {
+  class { 'ocf::packages::grafana::apt':
+    stage => first,
+  }
+
+  $canonical_url = $::host_env ? {
+    'dev'  => 'dev-prometheus.ocf.berkeley.edu',
+    'prod' => 'prometheus.ocf.berkeley.edu',
+  }
+
+  class { 'grafana':
+    install_method           => 'repo',
+
+    # Unfortunately, puppet-grafana requires that we specify a version number
+    # here.
+    version                  => '5.2.1',
+    manage_package_repo      => false,
+
+    cfg                      => {
```

Comment:
weird indentation

Reply:
I'm pretty sure the linter enforced this, but I can check again.